components/41-organisms/form/README.md

```diff
@@ -0,0 +1,91 @@
+# Form
+
+## When to use this component
+
+Use the form component to collect user input.
+
+## How it works
+
+### Form elements
+
+A form component contains form elements. Form elements are different types of input elements.
+ 
+The following form elements can be used in the form component:
+
+* <a href="{{path './formitem.html'}}">Input</a>
+* <a href="{{path './checkboxes.html'}}">Checkboxes</a>
+* <a href="{{path './radios.html'}}">Radios</a>
+* <a href="{{path './select.html'}}">Select</a>
+* Textarea <!-- @TODO Add link to the file upload component -->
```

Comment:
We have a textarea component, in formitem.html or form-item--textarea.html

Reply:
The textarea is a component that still needs to be splitted of the from formitem.html which should actually only be just (text) input.